snap/snapcraft.yaml

```diff
@@ -147,6 +150,8 @@ parts:
     organize:
       usr/share/doc/percona-backup-mongodb/copyright:
         licenses/LICENSE-percona-backup-mongodb
+      usr/share/doc/libldap-common/copyright:
+        licences/LICENSE-libldap-common
```

Comment:
I think this can be dropped, given those come from the ubuntu main archive

Reply:
I know that some other snaps do add all licenses in the snap, such as [charmed-postgresql-snap](https://github.com/canonical/charmed-postgresql-snap/blob/14/edge/snap/snapcraft.yaml).
I have no strong opinion on that so let's remove it if you say it's fine.